index.html

```diff
@@ -1,59 +1,284 @@
-                    <li><a class="nav-links black-text" href=""><i class="fas fa-envelop"></i></a></li>
-                </ul>
-                <div class="burger-parents">
-                    <div class="hamburger"></div>
-                    <div class="hamburger"></div>
-                    <div class="hamburger"></div>
-                </div>
-                
+      <header id="header">
+        <div class="logo blue-text"><a href="#">Joeveloper</a></div>
+        <nav class="nav-bar">
+          <ul id="navlink-ul">
+            <li><a class="nav-links black-text" href="">Portfolio</a></li>
+            <li><a class="nav-links black-text" href="">About</a></li>
+            <li><a class="nav-links black-text" href="">Contact</a></li>
+            <li>
+              <a class="nav-links black-text" href=""
+                ><i class="fas fa-envelop"></i
+              ></a>
+            </li>
+          </ul>
+          <div class="burger-parents">
+            <div class="hamburger"></div>
+            <div class="hamburger"></div>
+            <div class="hamburger"></div>
+          </div>
         </nav>
-    </header>
+      </header>
 
-    <section id="portfolio-main">
+      <section id="portfolio-main">
         <div id="div">
-            <div class="main-intro">
-                <div id="maintext-body">
-                    <h1 class="intro-heading .black-text">I'm Joeveloper</h1>
-                    <h1 class="intro-heading">Glad to see you!</h1>
-                    <p id="intro-para" class="black-text">I'm a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                    don’t hesitate to contact me.</p>
-                </div>
-            </div>
-            <div id="social-media">
-                <a href="#" id="social-para" class="social-blue">LET'S CONNECT</a>                    
-                <ul>
-                    <li><a target="_blank" class="social-tag" href="https://www.github.com/joeveoper"><img src="/images/twitter.png"></a></li>
-                    <li><a target="_blank" class="social-tag" href="https://www.linkedin.com/in/yinka-olatunji-83aa36b5/"><img src="/images/linkedin.png"></a></li>
-                    <li><a target="_blank" class="social-tag" href="https://www.twitter.com/joeveoper"><img src="/images/medium.png"></a></li>
-                    <li><a target="_blank" class="social-tag" href="https://www.medium.com/"><img src="/images/github.png"></a></li>
-                    <li><a target="_blank" class="social-tag" href="https://www.angellist.com/"><img src="/images/angellist.png"></a></li>
-                </ul>
+          <div class="main-intro">
+            <div id="maintext-body">
+              <h1 class="intro-heading .black-text">I'm Joeveloper</h1>
+              <h1 class="intro-heading">Glad to see you!</h1>
```

Comment:
For semantic reasons, I suggest you use just one header tag for your entire page😉 This will increase your accessibility score and will make screen readers appreciate you😉I can see you wanted to add a style to some of the text, but we can make it better this way😊 `<h1 class="intro-heading"><span class=".black-text">I'm Joeveloper</span> <br />Glad to see you!</h1>` something like this is more semantic😉